app/models/computer.rb

```diff
@@ -0,0 +1,2 @@
+class Computer < ApplicationRecord
+end
```

Comment:
Greetings, Are you agree to add these associations.
  has_many :favorites
  has_many :users, through: :favorites